maestro-ios-xctest-runner/build-maestro-ios-runner.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 set -euo pipefail
 
-if [ "$(basename "$PWD")" != "Maestro" ]; then
+if [ "$(basename "$PWD")" != "maestro" ]; then
```

Comment:
Except, the repo is Maestro, not maestro

<img width="354" height="135" alt="image" src="https://github.com/user-attachments/assets/50f90311-c302-4ec5-8a1b-04f24f3795c5" />

Perhaps a case insensitive match would be better?